modules/src/core/ics02_client/handler/create_client.rs

```diff
@@ -20,6 +22,8 @@ pub struct Result {
     pub client_type: ClientType,
     pub client_state: AnyClientState,
     pub consensus_state: AnyConsensusState,
+    pub processed_time: Timestamp,
+    pub processed_height: Height,
```

Comment:
I noticed these two fields are not used anywhere. Can we delete?

Reply:
This is a nice catch @adizere! This led me to realize that we're not initializing `ClientState` metadata in our impl. See https://github.com/cosmos/ibc-go/blob/main/modules/light-clients/07-tendermint/types/client_state.go#L187
Updated meta-issue #1696 with this. 👍